components/Hamburger/_hamburger.html

```diff
@@ -0,0 +1,61 @@
+            <div class="title">
+                <i class="fas fa-bars menu-bar"></i>
+                <h2 class="nav-title">My Website</h2>
+            </div>
+            <div class="search">
+                <span class="btn-search"><i class="fas fa-search"></i></span>
+                <input type="text" placeholder="Type to Search" class="input-search">
+            </div>
+            <div class="user-controls">
+                <button class="btn-login">Login</button>
+                <div class="btn-check">
+                    <i class="far fa-heart"></i>
+                    <span class="count">0</span>
+                </div>
+                <div class="btn-check">
+                    <i class="fas fa-shopping-cart"></i>
+                    <span class="count">0</span>
+                </div>
+            </div>
+        </div>
+        <div class="hamburger-menu">
+            <div class="profile-header">
+                <i class="fas fa-user"></i>
+                <i class="fas fa-times menu-bar-close"></i>
+            </div>
+            <div class="page-routes">
+                <ul>
+                    <li><a href="#">Home</a></li>
+                    <li><a href="#">Shop Now</a></li>
+                    <li><a href="#">Orders</a></li>
+                </ul>
+            </div>
+        </div>
+        <div class="content">
+            <p class="content-para">
+                Lorem ipsum dolor sit amet consectetur adipisicing elit. Labore provident modi iste culpa, nulla ipsum quos consequatur, non, sint quasi blanditiis perferendis voluptatum omnis minus facere dolorum? Perferendis natus veritatis corrupti est repellendus soluta asperiores officiis quibusdam error id facilis unde itaque voluptatem expedita vel, fugiat ipsam nostrum libero ipsum ut cum similique odio eveniet sit! Quibusdam laborum maxime accusamus. Aspernatur nisi quo, necessitatibus et natus numquam animi ullam ad delectus fugiat unde, accusantium sunt cumque. Sequi repudiandae maxime ducimus sed nesciunt alias harum qui aperiam sunt velit itaque, cum deleniti quos odio corporis dolores suscipit facilis eum eius dignissimos quas earum. Praesentium error perspiciatis officia, distinctio, ratione est provident corrupti ipsum expedita soluta ipsa, dicta consectetur ducimus sunt consequuntur fuga totam vitae dolores fugiat laborum. Saepe nemo quisquam nihil asperiores totam alias error ducimus voluptate, tenetur nisi qui cum delectus quia illum eligendi aliquam rem ullam magni eaque facere natus rerum deserunt sint dolore. Sint voluptatum vel sunt rem enim odio, reiciendis doloribus. Quod, officiis. Voluptas at cum consequatur tempora modi autem voluptates magni vero! Atque vel voluptas, fuga iusto voluptates maiores impedit, inventore vitae deleniti iure dolor nobis ipsum sed eaque illo facere adipisci earum voluptatum, quos minima.
+            </p>
+        </div>
+    </div>
+
+    <script src="./hamburger.js"></script>
+</body>
+
+</html>
```

Comment:
You could add an empty line at the end of the file to avoid such warnings from Github.

Reply:
Yes I would do that from my next project onwards. It would take a lot of time to edit all the files to have same indentation in all.